db/tags/utils.cc

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "db/tags/utils.hh"
+
+#include "db/tags/extension.hh"
+#include "schema_builder.hh"
+#include "schema_registry.hh"
+
+namespace db {
+
+const std::map<sstring, sstring>* get_tags_of_table(schema_ptr schema) {
```

Comment:
The commit message still refers to std::optional<std::ref>, please update it now that you have a pointer instead.